04_HowTos/SPARCL/How_to_use_SPARCL.html

```diff
@@ -7785,7 +7778,7 @@ <h4 id="We-now-create-a-SPARCL-client-instance:">We now create a SPARCL client i
 <div class="jp-InputArea jp-Cell-inputArea"><div class="jp-InputPrompt jp-InputArea-prompt">
 </div><div class="jp-RenderedHTMLCommon jp-RenderedMarkdown jp-MarkdownOutput" data-mime-type="text/markdown">
 <p><a class="anchor" id="auth_sparcl"></a></p>
-<h1 id="Authentication-for-SPARCL">Authentication for SPARCL<a class="anchor-link" href="#Authentication-for-SPARCL">¶</a></h1><p>All public SPARCL data sets can be accessed without explicitly logging in. However, some data sets are private and can only be accessed by authorized users. If you are an authorized user and wish to login to SPARCL, un-comment the cell below and enter your SSO user name and password.</p>
+<h1 id="Authentication-for-SPARCL">Authentication for SPARCL<a class="anchor-link" href="#Authentication-for-SPARCL">¶</a></h1><p>All public SPARCL data sets can be accessed without explicitly logging in. However, some data sets are private and can only be accessed by authorized users. If you are an authorized user and wish to login to SPARCL, un-comment the cell below and enter your NOIRLab CSDC SSO user name and password. If you need to create an account, sign-up at <a href="https://sso.csdc.noirlab.edu/">https://sso.csdc.noirlab.edu/</a> then email datalab-spectro@noirlab.edu with your First Name, Last Name, and Email Address (the same one you used to create your NOIRLab CSDC SSO account).</p>
```

Comment:
"If you need to create an account, sign-up at https://sso.csdc.noirlab.edu." is great but the second part could be misleading because we will already populate the list of authorized SPARCL users (unless there is an issue and a user registers for SSO with different credentials). So let's revise the rest to something like:  "If you encounter an issue, email datalab-spectro@noirlab.edu with your First Name, Last Name, and Email Address (the same one you used to create your NOIRLab CSDC SSO account)."